test/acqusition.test.mjs

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import assert from 'assert';
+import { describe, it } from 'node:test';
+import { classifyAcquisition } from '../src/acquisition.mjs';
+
+describe('classifyAcquisition', () => {
+  const testCases = [
+    { input: 'google', expected: 'paid:search:google', only: true },
+    { input: 'facebook', expected: ':social:facebook' },
+    { input: 'fb', expected: ':social:facebook' },
+    { input: 'bing', expected: 'paid:search:bing' },
+    { input: 'ig', expected: ':social:instagram' },
+    { input: 'meta', expected: ':social:facebook' },
+    { input: 'youtube', expected: ':video:youtube' },
+    { input: 'linkedin', expected: ':social:linkedin' },
+    { input: 'gmb', expected: '' },
+    { input: 'tiktok', expected: 'paid:video:tiktok', only: true },
```

Comment:
Is Tiktok really always paid? Can't you just share a tiktok link with someone just for the social aspect (i.e. Organic Social)

Reply:
Yes, Tiktok and Instagram are very hostile to users adding links, so its safe to assume that its paid.